src/app/cli/interpreter.hpp

```diff
@@ -36,7 +36,8 @@
 
 #include <map>
 
-#include "commissioner_app.hpp"
+#include <border_agent.hpp>
+#include <commissioner_app.hpp>
```

Comment:
```suggestion
#include "app/border_agent.hpp"
#include "app/commissioner_app.hpp"
```

Reply:
Thanks, will address this for all headers in separate PR.